bin/solr_init/create_core.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+INSTANCE_DIR=$SOLR_HOME/example/solr/$CKAN_SOLR_CORE
+cp -R $SOLR_HOME/example/solr/collection1 $SOLR_HOME/example/solr/$CKAN_SOLR_CORE
+cp $CKAN_SOLR_SCHEMA $SOLR_HOME/example/solr/$CKAN_SOLR_CORE/conf
+
+curl "http://localhost:8983/solr/admin/cores?action=CREATE&name=$CKAN_SOLR_CORE&instanceDir=$INSTANCE_DIR"
```

Comment:
you've got `localhost:8983` hard-coded here. Add another variable?

What do you think about using bash defaults at the top of these scripts for the variables they need? That sort of documents them and will make our circle.yml much smaller.

e.g.
```bash
CKAN_SOLR_CORE=${CKAN_SOLR_CORE:-ckan}
CKAN_SOLR_SCHEMA=${CKAN_SOLR_SCHEMA:-$HOME/ckan/ckan/config/solr/schema.xml}
SOLR_HOME=${SOLR_HOME:-$HOME/solr}
...
```